jscode.js

```diff
@@ -0,0 +1,22 @@
+var words = ['It', 'is', 'awfully', 'hard'];

```

Comment:
Use `let`, because you don't want to change these words in the future. It is an additional check that will help you to avoid bugs in the future.